src/Microsoft.AspNet.Identity/IIdentityKeyNormalizer.cs

```diff
@@ -4,15 +4,15 @@
 namespace Microsoft.AspNet.Identity
 {
     /// <summary>
-    ///     Used to normalize a user name
+    ///     Used to normalize keys for consistent lookups
     /// </summary>
-    public interface IUserNameNormalizer
+    public interface IIdentityKeyNormalizer
```

Comment:
We only have one other interface with identity and mostly do not, I will file a bug to rationalize, I think we should drop it from IIdentityMessageProvider.